examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 1742 (1.7K) [text/plain]\n",
+            "Saving to: ‘scp_to_manifest.py’\n",
+            "\n",
+            "scp_to_manifest.py  100%[===================>]   1.70K  --.-KB/s    in 0s      \n",
+            "\n",
+            "2020-05-11 02:14:48 (42.4 MB/s) - ‘scp_to_manifest.py’ saved [1742/1742]\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VgTR8CMlxu3p",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **SPEAKER RECOGNITION** \n",
+        "\n",
+        "Speaker Recognition (SR) is an broad research area which solves two major tasks: speaker identification (who is speaking?) and speaker verification (is the speaker who she claims to be?). In this work, we focmus on the far-field, text-independent speaker recognition when the identity of the speaker is based on how speech is spoken, not necessarily in what is being said. Typically such SR systems operate on unconstrained speech utterances, which are converted into vector of fixed length, called speaker embedding. Speaker embedding is also used in automatic speech recognition (ASR) and speech synthesis.\n",
+        "\n",
+        "As goal of most speaker related systems is to get good speaker level embeddings that could help distinguish from other speakers, we shall first train these embeddings in end-to-end manner optimizing the [QuatzNet](https://arxiv.org/abs/1910.10261) based encoder model on cross-entropy loss. We modify the original quartznet based decoder to get these fixed size embeddings irrespective of length of input audio. We employ mean and variance based statistics pooling method to grab these embeddings."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KzzOC5rpx9y6",
+        "colab_type": "text"
+      },
+      "source": [
+        "In this tutorial we shall first train these embeddings on speaker related datasets and then get speaker embeddings from a pretrained network for a new dataset. Since Google Colab has very slow read-write speeds I'll be demonstarting this tutorial using [an4](http://www.speech.cs.cmu.edu/databases/an4/). \n",
+        "\n",
+        "Instead if you'd like to try on a bigger dataset like [hi-mia](https://arxiv.org/abs/1912.01231) use the [get_hi-mia-data.py](https://github.com/NVIDIA/NeMo/blob/master/scripts/get_hi-mia_data.py) script to download the necessary files, extract them, also re-sample to 16Khz if any of the sample is not at 16Khz. This will take a while so grap a large coffee. We do also provide scripts to score these embeddings for a speaker-verification task for hi-mia dataset. To do that follow this detailed [tutorial](https://nvidia.github.io/NeMo/). "
```

Comment:
extract them, also re-sample to 16Khz if any of the sample is not at 16Khz. 
->
 extract them, also re-sample to 16Khz if any of these samples are not at 16Khz. 

` This will take a while so grap a large coffee. ` Please fix this sentence

Reply:
Done